src/Infrastructure/Form/Extension/ChannelTypeExtension.php

```diff
@@ -31,7 +42,48 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add('lowestPriceForDiscountedProductsCheckingPeriod', IntegerType::class, [
                 'label' => 'sylius_price_history.form.channel.period_for_which_the_lowest_price_is_calculated',
             ])
+            ->add('taxonsExcludedFromShowingLowestPrice', TaxonAutocompleteChoiceType::class, [
+                'label' => 'sylius_price_history.ui.taxons_for_which_the_lowest_price_is_not_displayed',
+                'required' => false,
+                'multiple' => true,
+            ])
         ;
+        $builder->setDataMapper($this);
+    }
+
+    public function mapDataToForms(mixed $viewData, \Traversable $forms): void
+    {
+        $this->propertyPathDataMapper->mapDataToForms($viewData, $forms);
+    }
+
+    public function mapFormsToData(\Traversable $forms, mixed &$viewData): void
+    {
+        /** @var \Traversable $traversableForms */
+        $traversableForms = $forms;
+
+        $forms = iterator_to_array($traversableForms);
```

Comment:
As the `$traversableForms` is not used anywhere else
```suggestion
        $forms = iterator_to_array($forms);
```